### ISSUE TITLE
Any plans to merge v0.5.1 PR (#7)?

### ISSUE THREAD

**Issue by DManowitz** [conda-forge/evotorch-feedstock#8]
### Comment:

@hadim, I see that PR #7 has been open more than 6 months now.  Any plans to merge it?

**Comment by hadim** [comment]
No, as I don't really maintain that feedstock anymore. Feel free to open a PR and also add you as a maintainer. I would be happy to merge those.

**Comment by hadim** [comment]
See https://github.com/conda-forge/evotorch-feedstock/pull/10
